### PR TITLE
feat: chart controllers Icon customization

### PR DESCRIPTION
**Pull request by asaharn**

### CODE REVIEW

File: pages/examples/index.html [hunk 1]
```diff
@@ -66,6 +66,25 @@
                         <!-- <img class="footerImage" src="images/logoGray.svg" title="Standalone Visualization"> -->
                     </div>
                 </div>
+                <div class='sampleContainer standalone'>
+                    <div class='sampleHeader'>
```
Comment (ag-ramachandran) [inline pages/examples/index.html:70]: Hope these PNG images do not have a copyrights issue :)

File: pages/examples/noauth/chartControllers.html [hunk 2]
```diff
@@ -0,0 +1,244 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Basic Charts</title>
```
Comment (ag-ramachandran) [inline pages/examples/noauth/chartControllers.html:5]: Basic charts with custom icons

File: src/UXClient/Components/LineChart/LineChart.ts [hunk 3]
```diff
@@ -1583,17 +1594,11 @@ class LineChart extends TemporalXAxisComponent {
             var self = this;
             this.hasStackedButton = true;
             this.stackedButton = this.chartControlsPanel.append("button")
-                .style("left", "60px")
```
Comment (ag-ramachandran) [inline src/UXClient/Components/LineChart/LineChart.ts:1586]: Any reason why this left style was removed ( ensuring some one else's alignment does not break )

File: src/UXClient/Components/LineChart/LineChart.ts [hunk 4]
```diff
@@ -1524,6 +1522,19 @@ class LineChart extends TemporalXAxisComponent {
...
+    public toggleAxis(): any {
+        console.log('toggleAxis- LineChart');
+        var self = this;
+        return function () {
+            self.overwriteSwimLanes();
+            self.render(self.data, { ...self.chartOptions, yAxisState: self.nextStackedState() }, self.aggregateExpressionOptions);
+            d3.select(this).attr("aria-label", () => self.getString("set axis state to") + ' ' + self.nextStackedState());
+            console.log('toggleAxis-'+ JSON.stringify(this));
+            setTimeout(() => (d3.select(self.renderTarget).node() as any).focus(), 200);
+            console.log('toggleAxis- LineChart Done');
```
Comment (ag-ramachandran) [inline src/UXClient/Components/LineChart/LineChart.ts:1534]: Is this console log needed ? perhaps for debugging ?

File: pages/examples/index.html [hunk 5]
```diff
@@ -66,6 +66,25 @@
                         <!-- <img class="footerImage" src="images/logoGray.svg" title="Standalone Visualization"> -->
                     </div>
                 </div>
+                <div class='sampleContainer standalone'>
```
[the remark this answers is not in the document]
Comment (asaharn) [reply pages/examples/index.html:69]: yes! screenshot of the customized chart controllers

File: pages/examples/index.html (hunk 1 above)
Comment (asaharn) [reply pages/examples/index.html:70]: Taken from https://www.svgrepo.com/. Free to download open licence svgs.

image

**Review by ag-ramachandran — APPROVED** [review]

**Pull request merged**